src/invidious/yt_backend/extractors.cr

```diff
@@ -129,6 +133,7 @@ private module Parsers
         live_now:           live_now,
         premium:            premium,
         premiere_timestamp: premiere_timestamp,
+        author_verified:    author_verified || false,
```

Comment:
you don't need the `|| false` fallback here either!

Reply:
Tried removing it and it seems to be required or at least at some point